doc/documentation.rst

```diff
@@ -1,738 +1,189 @@
+lectures on advanced topics available as well.
 
-**MNE basics**
-
-.. toctree::
-    :maxdepth: 1
 
-    tutorials/philosophy.rst
-    manual/cookbook.rst
-    whats_new.rst
-    python_reference.rst
-    glossary.rst
-    auto_examples/index.rst
-    generated/commands.rst
-    auto_tutorials/plot_configuration.rst
-    cited.rst
-    faq.rst
+.. _design_philosophy:
 
-**More help**
+MNE-Python design philosophy
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-- :ref:`Cite MNE <cite>`
-- `Mailing list <https://mail.nmr.mgh.harvard.edu/mailman/listinfo/mne_analysis/>`_ for analysis talk
-- `GitHub issues <https://github.com/mne-tools/mne-python/issues/>`_ for
-  requests and bug reports
-- `Gitter <https://gitter.im/mne-tools/mne-python>`_ to chat with devs
-
-.. raw:: html
-
-          </div>
-        </div>
-      </div>
-
-
-      <div class="panel panel-default">
-        <div class="panel-heading">
-          <h4 class="panel-title">
-            <a data-toggle="collapse" href="#collapse_data">Data structures and containers</a>
-          </h4>
-        </div>
-
-        <div id="collapse_data" class="panel-collapse collapse">
-          <div class="panel-body">
-
-.. toctree::
-    :maxdepth: 1
+**Interactive versus scripted analysis.** MNE-Python has some great interactive
+plotting abilities that can help you explore your data, and there are a few
+GUI-like interactive plotting commands (like browsing through the raw data and
+clicking to mark bad channels, or click-and-dragging to annotate bad temporal
+spans). But in general it is not possible to use MNE-Python to mouse-click your
+way to a finished, publishable analysis. MNE-Python works best when you
+assemble your analysis pipeline into one or more Python scripts. On the plus
+side, your scripts act as a record of everything you did in your analysis,
+making it easy to tweak/reproduce your analysis later and/or share it with
+others (including your future self).
 
-    auto_tutorials/plot_object_raw.rst
-    auto_tutorials/plot_object_epochs.rst
-    auto_tutorials/plot_object_evoked.rst
-    auto_tutorials/plot_object_source_estimate.rst
-    auto_tutorials/plot_info.rst
-    auto_tutorials/plot_object_annotations.rst
+**Integration with the scientific python stack.** MNE-Python also integrates
+well with other standard scientific python libraries. For example, MNE-Python
+objects underlyingly store their data in NumPy arrays, making it easy to apply
+custom algorithms or pass your data into one of `scikit-learn`_'s machine
+learning pipelines. MNE-Python's 2-D plotting functions also return matplotlib
+figure objects, and the 3D plotting functions return mayavi scenes, so you can
+customize your MNE-Python plots using any of matplotlib or mayavi's plotting
+commands. The intent is that MNE-Python will get most neuroscientists 90% of
+the way to their desired analysis goal, and other packages can get them over
+the finish line.
```

Comment:
I love this part. I think we should point to places that are clear examples of the integrations like [this one using sklearn's pipeline and function transformer](https://mne-tools.github.io/dev/auto_tutorials/plot_sleep.html#multiclass-classification-workflow-using-scikit-learn) 

Maybe we should also have an example of a custom plotting theme and how to apply it. (I think @larsoner did something like this)

And we should have several of those integration examples and collect them into a common place. So maybe we should keep this paragraph tight and point to another page where we have a show off gallery of integrations.

Reply:
>I love this part. I think we should point to places that are clear examples of the integrations like this one using sklearn's pipeline and function transformer

I agree. This is how I frame MNE at workshops.